modules/tensor_mechanics/src/materials/abaqus/AbaqusUMATStress.C

```diff
@@ -51,6 +51,9 @@ AbaqusUMATStress::validParams()
       "displaced mesh for computing displacements and quantities based on the deformed state.");
   params.addParam<UserObjectName>(
       "step_user_object", "The StepUserObject that provides times from simulation loading steps.");
+  params.addParam<Real>("euler_angle_1", 0.0, "Euler angle in direction 1.");
+  params.addParam<Real>("euler_angle_2", 0.0, "Euler angle in direction 2.");
+  params.addParam<Real>("euler_angle_3", 0.0, "Euler angle in direction 3.");
```

Comment:
```suggestion
  params.addParam<RealVectorValue>("euler_angles", "Euler angles corresponding to the Abaqus ORIENTATION keyword");
```

Reply:
My thinking was to list the Euler angles separately to make it consistent with [ComputeElasticityTensor](https://mooseframework.inl.gov/source/materials/ComputeElasticityTensor.html)

Reply:
Done